cmd/clusterctl/cmd/get_kubeconfig.go

```diff
@@ -59,6 +63,8 @@ func init() {
 		"Path to the kubeconfig file to use for accessing the management cluster. If unspecified, default discovery rules apply.")
 	getKubeconfigCmd.Flags().StringVar(&gk.kubeconfigContext, "kubeconfig-context", "",
 		"Context to be used within the kubeconfig file. If empty, current context will be used.")
+	getKubeconfigCmd.Flags().BoolVar(&gk.userKubeconfig, "system", true,
+		"If set to false, returns the kubeconfig of the system, else returns the user specific kubeconfig.")
```

Comment:
We should probably define what we mean by **system** kubeconfig somewhere. Perhaps in docs.

Reply:
What we are referring to here as the `system kubeconfig` is the kubeconfig used by CAPI to connect to child clusters.

Reply:
Yes. The one used by CAPI, or not-user kubeconfig is the system kubeconfig. The one currently provided by CAPI is system kubeconfig in this context.